Makefile

```diff
@@ -50,3 +50,11 @@ clear-cache:
 	docker rm -vf $$(docker ps -a -q) 2>/dev/null ; \
 	docker rmi -f $$(docker images -a -q) 2>/dev/null ; \
 	docker volume rm $$(docker volume ls -q) 2>/dev/null ; true
+
+experiment=*
```

Comment:
conventional style would be `EXPERIMENT := "*"`

Reply:
Some concerns at first since EXPERIMENT may be used by the fuzzbench experiment.
Later I found, EXPERIMENT is exactly the name we want actually. So we can use all capital letters here.